models/users/scratch/snowplow_mobile_users_aggs.sql

```diff
@@ -1,14 +1,14 @@
-{{ 
+{{
   config(
-    partition_by = {
+    partition_by = snowplow_utils.get_partition_by(bigquery_partition_by={
       "field": "start_tstamp",
       "data_type": "timestamp"
-    },
+    }, databricks_partition_by='start_tstamp'),
```

Comment:
`start_tstamp_date`

Also there's a question of should we even partition scratch tables, but maybe that's a bigger question that's worth having outside of this PR